packages/core/src/BinaryData/BinaryData.service.ts

```diff
@@ -51,6 +51,7 @@ export class BinaryDataService {
 		if (!manager) {
 			const { size } = await stat(filePath);
 			binaryData.fileSize = prettyBytes(size);
```

Comment:
perhaps we should re-use `fileSize` to store the raw value, and do the `prettyBytes` call in the frontend. I don't remember why I did it like this, but in retrospect, this feels wrong. any presentation related code should be completely in the frontend.